merged_schema/HPXMLMerged.xsd

```diff
@@ -2532,189 +2532,289 @@
 		</xs:choice>
 		<xs:attribute name="dataSource" type="DataSource"/>
 	</xs:complexType>
-	<xs:complexType name="Pools">
+	<xs:complexType name="PoolSpaType">
 		<xs:sequence>
-			<xs:element maxOccurs="unbounded" name="Pool">
+			<xs:group ref="SystemInfo"/>
+			<xs:element minOccurs="0" name="Type" type="PoolType">
+				<xs:annotation>
+					<xs:documentation>Indicates if the pool is above or below ground.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="Volume" type="Volume">
+				<xs:annotation>
+					<xs:documentation>[gal] Volume of pool.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="MonthsPerYearofOperation" type="MonthsPerYear">
+				<xs:annotation>
+					<xs:documentation>Months per year pool is in operation.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="ReturnPipeDiameter" type="LengthMeasurement">
+				<xs:annotation>
+					<xs:documentation>[in]</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="SuctionPipeDiameter" type="LengthMeasurement">
+				<xs:annotation>
+					<xs:documentation>[in]</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="FilterType" type="PoolFilterType">
+				<xs:annotation>
+					<xs:documentation>Type of filter used, if any.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="PoolPumps">
```

Comment:
Should this be renamed to just `Pumps` since it shows up for `StationarySpa` too? Obviously a breaking change...